source-formatter.properties

```diff
@@ -382,6 +382,7 @@
         modules/apps/petra/petra-content,\
         modules/apps/petra/petra-doulos,\
         modules/apps/petra/petra-encryptor,\
+        modules/apps/petra/petra-http-invoker,\
```

Comment:
@julschong The change is not consistent with your commit message.

Reply:
@tinatian 
sorry for the mix up.